mitodl/mit-open
# Shanbady/browse by topics UI fix

### What are the relevant tickets?

Closes [4475](https://github.com/mitodl/hq/issues/4475)

### Description (What does it do?)
Fixes visual issues with topics section on the homepage

### Screenshots (if appropriate):

<img width="676" alt="Screenshot 2024-06-13 at 4 42 53 PM" src="https://github.com/mitodl/mit-open/assets/196425/7f15bf21-daef-4b42-b63c-9bc4b38b3e66">


### How can this be tested?
1. Checkout this branch. 
2. make sure you have topics loaded up. 
3. visit home page and scroll down to topics section.
4. make sure it matches [the mocks](https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?page-id=107%3A2479&node-id=3852-51162&viewport=-9072%2C-80%2C1.96&t=ipWVEEPGFXgjOvwu-1&scaling=min-zoom&starting-point-node-id=3852%3A51162)
